docfx_yaml/extension.py

```diff
@@ -758,12 +758,107 @@ def _extract_docstring_info(summary_info, summary, name):
             words = []
         else:
             words.append(word)
-    
+
         index += 1
 
     return top_summary
 
 
+def reformat_summary(summary: str) -> str:
```

Comment:
Consider moving this down below the other "exported" reformat function.

Reply:
Done.